broken-python/plan.sh

```diff
@@ -0,0 +1,33 @@
+pkg_name=broken-python
+pkg_description="Example of broken thread usage in core/python."
+# $HAB_ORIGIN overrides pkg_origin.
+pkg_origin=origin
+pkg_version=0.1.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_source=https://github.com/habitat-sh/core-plans
+pkg_deps=(core/gcc-libs core/glibc core/python)
```

Comment:
I pretty should this just should be `pkg_deps=(core/python)`. I added `core/gcc-libs core/glibc` as a brute force attemp to solve the problem - to no avail.